src/main/java/com/google/step/servlets/JsonFormatter.java

```diff
@@ -143,6 +144,8 @@ public static String getUserJson(
     userMap.put("bio", user.bio);
     if (user.photoBlobKey.isPresent()) {
       userMap.put("picture", "/api/images/" + user.photoBlobKey.get());
```

Comment:
```suggestion
      userMap.put("picture", getImageUrl(user.photoBlobKey.get()));
```
Can change to use the function instead of hardcoding the url everywhere.